lib/vmpooler/pool_manager.rb

```diff
@@ -1,12 +1,17 @@
 module Vmpooler
   class PoolManager
-    def initialize(config, logger, redis, graphite=nil)
+    def initialize(config, logger, redis, graphite=nil, statsd=nil)
```

Comment:
(Also, please throw some spaces between the value and the name in the parameter assignments. Whitespace is free.)